Gemfile

```diff
@@ -83,7 +83,7 @@ gem 'sitemap_generator'
 gem 'blacklight-gallery', '>= 0.3.0'
 gem 'blacklight-hierarchy'
 gem 'blacklight-oembed', '>= 0.1.0'
-gem 'blacklight_range_limit', '~> 7.4'
+gem 'blacklight_range_limit', github: 'projectblacklight/blacklight_range_limit', branch: 'master'
```

Comment:
A new release is being tracked in our backlog. Before doing those mechanics, I wanted to make sure we were good to go.